cf-oscore/src/test/java/org/eclipse/californium/oscore/OSCoreObserveTest.java

```diff
@@ -0,0 +1,348 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Institute for Pervasive Computing, ETH Zurich and others.
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * This test class is based on org.eclipse.californium.core.test.SmallServerClientTest
+ * 
+ * Contributors:
+ *    Matthias Kovatsch - creator and main architect
+ *    Martin Lanter - architect and re-implementation
+ *    Dominique Im Obersteg - parsers and initial implementation
+ *    Daniel Pauli - parsers and initial implementation
+ *    Kai Hudalla - logging
+ *    Achim Kraus (Bosch Software Innovations GmbH) - use CoapNetworkRule for
+ *                                                    setup of test-network
+ *    Achim Kraus (Bosch Software Innovations GmbH) - destroy server after test
```

Comment:
I would prefer, that 
``` 
/*******************************************************************************
 * Copyright (c) 2018 RISE SICS and others.
 * 
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * and Eclipse Distribution License v1.0 which accompany this distribution.
 * 
 * The Eclipse Public License is available at
 *    http://www.eclipse.org/legal/epl-v10.html
 * and the Eclipse Distribution License is available at
 *    http://www.eclipse.org/org/documents/edl-v10.html.
 * 
 * Contributors: 
 *    Rikard Höglund (RISE SICS) - testing OSCORE Observe messages
```

Is used as header including your comment
```
This test class is based on org.eclipse.californium.core.test.SmallServerClientTest
```

Reply:
Sure, I have now update the file accordingly in commit 2e5ab13698dbb2495c8021975cc5a9a8fb70fa35.

Reply:
:-) I was wrong, we have 2019, sorry :-).
could you please adjust it again?

Reply:
> :-) I was wrong, we have 2019, sorry :-).
> could you please adjust it again?

Sure, it's been updated now :) See commit b0ec659f2ed558535aafea0e01da4f3b92652f30